### DISCUSSION TITLE
Only Delay Once When a Link Leads to a Redirect

### DISCUSSION THREAD

**Discussion by azdle** [proginosko/LeechBlockNG#361] [Ideas]
Right now, if a block set applies to both the pre-redirect and post-redirect URLs I have to sit through my delay twice.

For example:

I have a blockset with the regex: `^https://([^.]*.)reddit\.com(/r/[^/]*)?/?$`

If I go to `https://www.reddit.com/r/3dprinting/`, I get the delay page. But, then after the delay it loads and gets a redirect to `https://www.reddit.com/r/3Dprinting/` (capital d) and that triggers the block set again and I have to wait through my delay again.

**Comment by proginosko** [comment]
Workaround would be to select "Block only first accessed page of site when delaying page is used".
